third-culture-software/bhima
# Update RUMER report

Updates the stock RUMER report

Closes https://github.com/IMA-WorldHealth/bhima/issues/6153

Includes these fixes:

- Updated English description to:  Drug Usage and Entries Report
- Modified report title to include "RUMER" and description on the same line
- Minor formatting adjustments
- Improved label wording for "Include daily balances" option
- Fixed logic for "Exclude inventories without movements and out of stock".  Now it will exclude any stock that has no movements for the month AND is out of stock.  Also notice that negative values are treated as zero now.

**TESTING**
- Use the Vanga dataset
- Stock > Reports > RUMER
- Try the various report options
- To verify the logic for the exclude option, try PHARMACIE USAGE, FY 2021, February 2021
   - Without the exclude option checked, look for entries for 
      - Captopril (Tablets USP) 25 mg Ces
      - Vitamine E de 200 mg Ces  (near the end)
   - Without the option, you should see both of these
   - With the exclude option checked, these should not appear
